.github/workflows/build-test.yaml

```diff
@@ -11,7 +11,7 @@ on:  # yamllint disable-line rule:truthy
     branches:
       - "*"
 env:
-  GO_VERSION: "~1.19"
+  GO_VERSION: "~1.20"
```

Comment:
move to 1.21 now that its out?

Reply:
we usually wait a little bit right? I figured we'd do spicedb first (I would want to move to the new stdlb slices/maps libs where possible too)

Reply:
K